src/modules/Modal/Modal.js

```diff
@@ -53,15 +57,21 @@ class Modal extends Component {
     /** Whether or not the Modal should close when the document is clicked. */
     closeOnDocumentClick: PropTypes.bool,
 
+    /** Simple text content for the Modal. */
+    content: customPropTypes.itemShorthand,
```

Comment:
As we update propTypes, we should also update the typings in the same directory.  Example, `Modal.d.ts`.